workspaces/sonarqube/packages/backend/src/index.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Hi!
+ *
+ * Note that this is an EXAMPLE Backstage backend. Please check the README.
+ *
+ * Happy hacking!
+ */
+
+import { createBackend } from '@backstage/backend-defaults';
+
+const backend = createBackend();
+
+backend.add(import('@backstage/plugin-app-backend/alpha'));
+backend.add(import('@backstage/plugin-proxy-backend/alpha'));
+
+// auth plugins
+backend.add(import('@backstage/plugin-auth-backend'));
+// See https://backstage.io/docs/backend-system/building-backends/migrating#the-auth-plugin
+// use actual configured github auth to be able to test github-actions integration is working as expected
+backend.add(import('./modules/auth'));
```

Comment:
We want to just use Guest auth here, that's sort of the universal path for these examples apps and local development.